File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.py
class Solution:
    def targetIndices(self, nums: List[int], target: int) -> List[int]:
        nums.sort()
        i=0
        result=[]
        while i<len(nums) and nums[i]<=target:
            if nums[i]<target:
                i+=1
            elif nums[i]==target:
                result.append(i)
                i+=1
            else:break
        return result
            

File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var targetIndices = function(nums, target) {
    nums.sort(numCompare)
    let i=0
    let result=[]
    while(nums[i]<=target){
        if(nums[i]<target){
            i++
        }
        else if(nums[i]===target){
            result.push(i)
            i++
        }else break
    }
    return result 
    
   
};
function numCompare(num1,num2){
    return num1-num2
}